comp/api/api/apiimpl/internal/agent/agent.go

```diff
@@ -28,11 +28,13 @@ import (
 	"github.com/DataDog/datadog-agent/cmd/agent/common/signals"
 	"github.com/DataDog/datadog-agent/cmd/agent/gui"
 	"github.com/DataDog/datadog-agent/comp/aggregator/demultiplexer"
+	"github.com/DataDog/datadog-agent/comp/api/api"
 	"github.com/DataDog/datadog-agent/comp/api/api/apiimpl/response"
 	"github.com/DataDog/datadog-agent/comp/collector/collector"
 	"github.com/DataDog/datadog-agent/comp/core/autodiscovery"
 	"github.com/DataDog/datadog-agent/comp/core/autodiscovery/autodiscoveryimpl"
-	"github.com/DataDog/datadog-agent/comp/core/flare"
+
+	// "github.com/DataDog/datadog-agent/comp/core/flare"
```

Comment:
Same about deleting not adding comments 😄

Reply:
removed in https://github.com/DataDog/datadog-agent/pull/24045/commits/dd24bb32483d68bc40fbebc5a077395d8fa2a4a4